src/core/packages/saved-objects/base-server-mocks/src/saved_objects_type_registry.mock.ts

```diff
@@ -33,6 +37,7 @@ const createRegistryMock = (): jest.Mocked<
 
   mock.getVisibleTypes.mockReturnValue([]);
   mock.getAllTypes.mockReturnValue([]);
+  mock.getLegacyTypes.mockReturnValue(REMOVED_TYPES);
```

Comment:
Are we using this info in any tests?, if not we can simply return `[]` IMO, no need to add coupling / knowledge about the REMOVED_TYPES here either.